src/components/binding/binding-loader.js

```diff
@@ -113,6 +118,29 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, fileName,
 };
 
 /**
+ * Loads the bed data of a single binding track.
+ * @param {string} bedName Bed name of the bed binding track.
+ * @param {string} chr Chromosome.
+ * @param {number|undefined} xl Left coordinate of the query range.
+ *   If null, use the leftmost coordinate of the track.
+ * @param {number|undefined} xr Right coordinate of the query range.
+ *   If null, use the rightmost coordinate of the track.
+ */
+genotet.BindingLoader.prototype.loadBed = function(bedName, chr, xl, xr) {
+  var params = {
+    type: 'bed',
+    bedName: bedName,
```

Comment:
Are you sure?
1) I didn't you see you change the server code regarding using 'bedName' instead of 'fileName'.
2) If we have switched to use fileName as ID, why would you revoke it?